tests/std/tests/P0429R9_flat_map/test.cpp

```diff
@@ -418,8 +418,16 @@ void test_insert() {
     assert(res3.first->second == 'w');
     assert(res3.second);
 
-    assert(check_key_content(fm, {10, 90}));
-    assert(check_value_content(fm, {'m', 'w'}));
+    flat_map<int, char>::iterator res4 = fm.insert(fm.cbegin(), std::pair<int, char>{30, 'c'});
+    assert(res4->first == 30);
+    assert(res4->second == 'c');
+
+    flat_map<int, char>::iterator res5 = fm.insert(fm.cbegin(), std::pair<char, char>{static_cast<char>(40), 'd'});

```

Comment:
The previous 3 insertions said `const auto` to store the returned iterators, and it would be reasonable to be consistent here. Ditto below.

(It's fine to explicitly say `std::pair<int, char>` and `std::pair<char, char>` to emphasize the types being inserted, even though `std::pair` CTAD would give the same result.)